docker/Dockerfile.core-command

```diff
@@ -7,9 +7,19 @@
 #
 
 FROM golang:1.9-alpine AS builder
+ENV GOPATH=/go
+RUN apk --update add \
+    make \
+    git \
+    ca-certificates \
+    glide \
+    zeromq-dev \
+    libsodium-dev \
+    pkgconfig \
+    build-base
+RUN go get github.com/edgexfoundry/edgex-go
```

Comment:
Nope, please do not do this. Why would you be getting the code from the GitHub, hen you have one locally?

This:
1. Prevents developers to build the Docker containers and test the code during the development
2. Breaks the CI - how would you build the version x.y.z with your approach? You are obviously always building them master in your Docker

Reply:
Thanks.  I've removed this